### PR TITLE
2

### PR DESCRIPTION
**Pull request by marcbloc**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+	<meta charset="UTF-8">
+	<link rel="stylesheet" type="text/css" href="css/style.css"/>
+	<title>Carousel</title>
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
```
Comment (matwjablonski) [inline index.html]: move script to the end of file

File: index.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	</head>
```
Comment (matwjablonski) [inline index.html]: align tag

File: js/scripts.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+$(function(){
+	var carouselList = $("#carousel ul");
+	setInterval(changeSlide, 3000);
+	function changeSlide() {
+		carouselList.animate({'marginLeft':-400}, 500, moveFirstSlide);
+}
+function moveFirstSlide(){
```
Comment (matwjablonski) [inline js/scripts.js]: add empty line before

File: js/scripts.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+$(function(){
+	var carouselList = $("#carousel ul");
+	setInterval(changeSlide, 3000);
+	function changeSlide() {
```
Comment (matwjablonski) [inline js/scripts.js]: align function

**Review by matwjablonski — CHANGES_REQUESTED** [review]

**Comment by marcbloc** [comment]
corrected

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**